client/src/pages/people/Form.js

```diff
@@ -117,8 +117,8 @@ class BasePersonForm extends ValidatableFormWrapper {
 		const { currentUser } = this.props
 		const isAdmin = currentUser && currentUser.isAdmin()
 		const isSelf = Person.isEqual(currentUser, person)
-		// status can only be changed by admins (but not of self!)
-		const disableStatusChange = !isAdmin || isSelf
+		// anyone can change status to INACTIVE, only admins can change back to ACTIVE (but nobody can change status of self!)
```

Comment:
It will be clearer if the comment says: `anyone with editing permission can change status to INACTIVE, only ...`